src/IonText.ts

```diff
@@ -267,51 +267,44 @@ function toCharCodes(s: string) {
   return charCodes;
 }
 
-function unicodeEscape(codePoint: number) : number[] {
-  let prefix: number[] = [CharCodes.BACKSLASH, CharCodes.LOWERCASE_U];
-  let hexEscape: string = codePoint.toString(16);
-  while (hexEscape.length < 4) {
-    hexEscape = "0" + hexEscape;
-  }
-  return prefix.concat(toCharCodes(hexEscape));
+function hexEscape(codePoint: number) : number[] {
+    let prefix: number[] = [CharCodes.BACKSLASH, CharCodes.LOWERCASE_X];
+    let hexEscape: string = codePoint.toString(16);
+    while (hexEscape.length < 2) {
+        hexEscape = "0" + hexEscape;
+    }
+    return prefix.concat(toCharCodes(hexEscape));
 }
 
-export let ClobEscapes : EscapeIndex = {};
-ClobEscapes[CharCodes.NULL] = backslashEscape("0");
-ClobEscapes[CharCodes.BELL] = backslashEscape("a");
-ClobEscapes[CharCodes.BACKSPACE] = backslashEscape("b");
-ClobEscapes[CharCodes.HORIZONTAL_TAB] = backslashEscape("t");
-ClobEscapes[CharCodes.LINE_FEED] = backslashEscape("n");
-ClobEscapes[CharCodes.VERTICAL_TAB] = backslashEscape("v");
-ClobEscapes[CharCodes.FORM_FEED] = backslashEscape("f");
-ClobEscapes[CharCodes.CARRIAGE_RETURN] = backslashEscape("r");
-ClobEscapes[CharCodes.DOUBLE_QUOTE] = backslashEscape('"');
-ClobEscapes[CharCodes.SINGLE_QUOTE] = backslashEscape("'");
-ClobEscapes[CharCodes.FORWARD_SLASH] = backslashEscape("/");
-ClobEscapes[CharCodes.QUESTION_MARK] = backslashEscape("?");
-ClobEscapes[CharCodes.BACKSLASH] = backslashEscape("\\");
-
-    function unicodeEscapes(escapes: EscapeIndex, start: number, end?: number) {
-        if (end === undefined) {
-            escapes[start] = unicodeEscape(start);
-        } else {
-            for (let i: number = start; i < end; i++) {
-                escapes[i] = unicodeEscape(i);
-            }
+function hexEscapes(escapes: EscapeIndex, start: number, end?: number) {
```

Comment:
Could you make the name more descriptive (maybe `populateHexEscapeIndex`?) or add a comment to that effect?

Reply:
Renamed to `populateWithHexEscapes`